app/Http/Controllers/Auth/LogoutController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Session;
+use Inertia\Inertia;
+
+class LogoutController extends Controller
+{
+    public function logout(Request $request)
+    {
+        Auth::logout();
+        $request->session()->invalidate();
+        $request->session()->regenerateToken();
+
+        Session::flash("success", "Logout Complite.");
+
+        return Inertia::location("/");
```

Comment:
```suggestion
        return redirect()->route("login");
```
That should be enough.